addons/auth_totp/data/mail_templates.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="mail_template_totp_invite" model="mail.template">
+        <field name="name">TOTP on User: Invite by email</field>
+        <field name="model_id" ref="base.model_res_users" />
+        <field name="subject">Invitation to activate 2-Step authentication on your Odoo account</field>
+        <field name="email_to">${(object.partner_id.email_formatted or object.email) |safe}</field>
+        <field name="partner_to">${object.partner_id.id}</field>
```

Comment:
You should probably not set both email_to and partner_to, risk is that you send email twice :D .

Reply:
To be more complete: check in existing templates, but globally email_to should have a value if object.partner_id is void, to avoid having both a partner and its email in the recipients, as it could be sent twice